mapiproxy/servers/default/emsmdb/oxcfxics.c

```diff
@@ -3164,6 +3202,18 @@ _PUBLIC_ enum MAPISTATUS EcDoRpc_RopSyncOpenCollector(TALLOC_CTX *mem_ctx,
 	synccontext_object->object.synccontext->request.contents_mode = (mapi_req->u.mapi_SyncOpenCollector.IsContentsCollector != 0);
 	handles[mapi_repl->handle_idx] = synccontext_rec->handle;
 
+	/* Create an idset to store the involved fmids in the upload
+	   operations ids generated by these operations and current
+	   change number to retrieve only state after updating
+	   uploaded changes */
+	synccontext_object->object.synccontext->involved_fmids = RAWIDSET_make(synccontext_object, false, false);
```

Comment:
Add a check on RAWIDSET_make to check if involved_fmids is NULL or not. RAWIDSET_make DOES NOT check talloc retval at the moment but will when I finish review the libmapi/idset.c memory allocation. This will avoid one line patch. Thanks!